Libplanet/Net/NetMQTransport.cs

```diff
@@ -560,6 +562,8 @@ public async Task CheckAllPeersAsync(CancellationToken cancellationToken, TimeSp
 
         private void ReceiveMessage(object sender, NetMQSocketEventArgs e)
         {
+            LastMessageTimestamp = DateTimeOffset.UtcNow;
```

Comment:
Was this intended to set `LastMessageTimestamp` even an invalid message is received?

Reply:
It wasn't. I'll fix it.

Reply:
Amended. d25f3d80d16eed2eb3138738a44a5d750365e2e7